creator-node/src/routes/files.js

```diff
@@ -12,6 +12,188 @@ const { authMiddleware, syncLockMiddleware, triggerSecondarySyncs } = require('.
 const { getIPFSPeerId, rehydrateIpfsFromFsIfNecessary, ipfsSingleByteCat } = require('../utils')
 const ImageProcessingQueue = require('../ImageProcessingQueue')
 
+// Helper method to stream file from file system on creator node
+const streamFromFileSystem = async (req, res, path) => {
+  try {
+    // If file cannot be found on disk, throw error
+    if (!fs.existsSync(path)) {
+      throw new Error('File could not be found on disk.')
+    }
+
+    // Stream file from file system
+    let fileStream
+
+    // If a range header is present, use that to create the readstream
+    // otherwise, stream the whole file.
+    const range = req.range()
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Range_requests#Partial_request_responses

I should be sending a 206 probably, updating that.

Our clients won't be using this, but the API layer will. Use cases look like this https://stackoverflow.com/a/22031835/11435157

Per suggestion of Chris, most of the users of this will be doing range requests (so they don't have to wait on the full content). This is what fanburst did, at least to support this use case.